packs/mailgun/actions/send_email.py

```diff
@@ -0,0 +1,31 @@
+import httplib
+
+import requests
+
+from st2actions.runners.pythonrunner import Action
+
+__all__ = [
+    'SendEmailAction'
+]
+
+SEND_EMAIL_API_URL = 'https://api.mailgun.net/v2/%(domain)s/messages'
+
+
+class SendEmailAction(Action):
```

Comment:
Questions for myself here... Let's say I have an integration with another Mail sender (Postmark, Amazon SES, whatever), and the pack action also uses the same class name. What happens? Should we care about namespacing here?